repository-data/development/src/main/resources/hcm-content/content/documents/bsh/sandbox.yaml

```
@@ -5559,6 +5551,660 @@
+              hippo:availability: []
+              hippostd:holder: admin
+              hippostd:retainable: false
+              hippostd:state: draft
+              hippostdpubwf:createdBy: admin
+              hippostdpubwf:creationDate: 2025-07-18T20:18:52.551+01:00
+              hippostdpubwf:lastModificationDate: 2025-07-22T14:22:53.970+01:00
+              hippostdpubwf:lastModifiedBy: admin
+              hippotranslation:id: 4af954ca-785c-49a8-b8d6-fcef0cab8818
+              hippotranslation:locale: en
+              visitscotland:features: [online-booking-process, pricing-funcionality,
+                cloud-based-updates, social-media, ota-apis, payment-processing, online-booking,
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 39 but found 16

(indentation)